src/test_utils.jl

```diff
@@ -13,7 +13,7 @@ function test_flatten_interface(x::T) where {T}
     return nothing
 end
 
-function test_parameter_interface(x::AbstractParameter)
```

Comment:
why do you need to relax this constraint? do you expect to use this function on non-AbstractParameter

Reply:
Yeah, I mean we can ask for the `value` of anything, so it seemed like a bit of a silly constraint.